Makefile

```diff
@@ -119,7 +129,7 @@ lint: go-lint shell-lint
 #      --mem-profile-path string   Path to memory profile output file
 # to debug big allocations during linting.
 .PHONY: go-lint
-go-lint: check-git deps $(GOLANGCI_LINT) $(FAILLINT) $(MISSPELL)
+go-lint: check-git deps check-config-docs $(GOLANGCI_LINT) $(FAILLINT) $(MISSPELL)
```

Comment:
We should not be mixing targets - checking the docs is not linting. Please add it as a separate CI job

Reply:
Ok!